spec/features/breadcrumbs_spec.rb

```diff
@@ -13,6 +13,11 @@
     @user_practice2 = Practice.create!(name: 'Another Best Practice', user: @user, initiating_facility: 'vc_0508V', tagline: 'Test tagline 2')
     login_as(@user, :scope => :user, :run_callbacks => false)
     PracticePartnerPractice.create!(practice_partner: @pp, practice: @user_practice)
+    @page_group = PageGroup.create!(name: 'programming', description: 'Pages about programming go in this group.')
+    @page_group2 = PageGroup.create!(name: 'test', description: 'Pages about tests go in this group.')
+    @page = Page.create!(title: 'Test', description: 'This is a test page', slug: 'home', page_group: @page_group)
```

Comment:
for these pages that are created we want to add `published: Time.now` so it is viewable :)

Reply:
Thx Elisa - I have updated that code for the Page.Create methods.  Please test on 1704 branch only...  everything for 1790 is now in 1704.